src/trans-netlist/trans_trace.cpp

```diff
@@ -109,20 +133,30 @@ void show_trans_state(
       std::cout << "?";
     else
       std::cout << from_expr(ns, symbol.name, rhs);
-    
-    if(rhs.type().id()==ID_unsignedbv ||
-       rhs.type().id()==ID_signedbv ||
-       rhs.type().id()==ID_bv)
+
+    if(rhs.type().id() == ID_unsignedbv || rhs.type().id() == ID_signedbv)
     {
       std::size_t width=to_bitvector_type(rhs.type()).get_width();
-      
+
+      if(width >= 2 && width <= 32 && rhs.id() == ID_constant)
```

Comment:
I only realize now that there always were those conditions: why those width bounds?

Reply:
Rather arbitrary, I suppose. If it's just one bit, then the binary output doesn't add anything over the decimal one.

In Verilog, you might see really large registers (say 1000 bits), which is why there's an upper limit. Perhaps 32 is a bit small. Another option could be to switch to hex output beyond a certain size.